src/Symfony/Component/ExpressionLanguage/Parser.php

```diff
@@ -330,7 +330,7 @@ public function parsePostfixExpression($node)
                     throw new SyntaxError('Expected name', $token->cursor);
                 }
 
-                $arg = new Node\ConstantNode($token->value);
+                $arg = new Node\NameNode($token->value);
```

Comment:
Yes it's a BC break. I created [a melody script](https://gist.github.com/lyrixx/d9e7f17c86d1ed8ae3869f23bbb254c8) to reproduce it.

@fabpot it's because of this change your gist your sent me yesterday does not work anymore.